a3p-integration/proposals/n:upgrade-next/test/initial.test.js

```diff
@@ -11,7 +11,7 @@ const expectedVatDetails = {
   orchestration: { incarnation: 2 },
   provisionPool: { incarnation: 2 },
   transfer: { incarnation: 4 },
-  walletFactory: { incarnation: 9 },
+  walletFactory: { incarnation: 10 },
```

Comment:
I'm surprised we're still upgrading the wallet factory in the chain software

Reply:
That's a a kernel concern, orthogonal to this PR, right?